Source/ios/Storyboard/UIResponder+TyphoonOutletTransfer.m

```diff
@@ -0,0 +1,126 @@
+////////////////////////////////////////////////////////////////////////////////
+//
+//  TYPHOON FRAMEWORK
+//  Copyright 2015, Typhoon Framework Contributors
+//  All Rights Reserved.
+//
+//  NOTICE: The authors permit you to use, modify, and distribute this file
+//  in accordance with the terms of the license agreement accompanying it.
+//
+////////////////////////////////////////////////////////////////////////////////
+
+#import "UIResponder+TyphoonOutletTransfer.h"
+#import "NSLayoutConstraint+TyphoonOutletTransfer.h"
+#import <objc/runtime.h>
+
+@implementation UIResponder (TyphoonOutletTransfer)
+
+- (void)transferFromView:(UIView *)view
```

Comment:
Maybe I got a little excited about the SRP breaks , but it seems to me that implementation of method has different levels abstraction under method name. It's my opinion :)